api-tests/features/cimit/alternate-doc.feature

```diff
@@ -1,241 +1,275 @@
+      Examples:
+        | initial-cri    | initial-invalid-doc                        | no-match-page                            | mitigating-cri | mitigating-doc               | prove-identity-no-other-photo-id-context |
+        | ukPassport     | kenneth-passport-needs-alternate-doc       | pyi-passport-no-match-another-way        | drivingLicence | kenneth-driving-permit-valid | drivingLicence                           |
+        | drivingLicence | kenneth-driving-permit-needs-alternate-doc | pyi-driving-licence-no-match-another-way | ukPassport     | kenneth-passport-valid       | passport                                 |
 
-  Scenario: Returns P0 when user continues to service from prove-identity-no-other-photo-id page during CI mitigation
-    When I submit a 'ukPassport' event
-    Then I get a 'ukPassport' CRI response
-    When I submit 'kenneth-passport-needs-alternate-doc' details to the CRI stub
-    Then I get a 'pyi-passport-no-match-another-way' page response
-    When I submit a 'next' event
-    Then I get a 'drivingLicence' CRI response
-    When I call the CRI stub and get an 'access_denied' OAuth error
-    Then I get a 'prove-identity-no-other-photo-id' page response with context 'drivingLicence'
-    When I submit a 'returnToRp' event
-    Then I get an OAuth response
-    When I use the OAuth response to get my identity
-    Then I get a 'P0' identity
+    Scenario: Returns P0 when user continues to service from prove-identity-no-other-photo-id page during CI mitigation
+      When I submit a 'ukPassport' event
+      Then I get a 'ukPassport' CRI response
+      When I submit 'kenneth-passport-needs-alternate-doc' details to the CRI stub
+      Then I get a 'pyi-passport-no-match-another-way' page response
+      When I submit a 'next' event
+      Then I get a 'drivingLicence' CRI response
+      When I call the CRI stub and get an 'access_denied' OAuth error
+      Then I get a 'prove-identity-no-other-photo-id' page response with context 'drivingLicence'
+      When I submit a 'returnToRp' event
+      Then I get an OAuth response
+      When I use the OAuth response to get my identity
+      Then I get a 'P0' identity
+
+  Rule: Existing identity
+    Scenario: Mitigating when a user already has an identity should be subject to a COI check
+      Given the subject already has the following credentials
+        | CRI        | scenario               |
+        | ukPassport | kenneth-passport-valid |
+        | address    | kenneth-current        |
+        | fraud      | kenneth-score-2        |
+        | kbv        | kenneth-score-2        |
+      And the subject has the following CIs
+        | code                | document                                           |
+        | NEEDS-ALTERNATE-DOC | drivingPermit/GB/DVLA/DECER607085K98FGA/2005-02-02 |
```

Comment:
Do you think it's worth running through the entire journey here (i.e. the change of details -> D02 -> return), to record the real-life scenario that might encounter this?

Reply:
Yeah, I suppose it might be - will fix it up